Doc/library/nntplib.rst

```diff
@@ -14,6 +14,20 @@
 .. deprecated:: 3.11
    The :mod:`nntplib` module is deprecated (see :pep:`594` for details).
 
+.. testsetup::
+
+   import warnings
+   with warnings.catch_warnings():
+       warnings.simplefilter('ignore', category=DeprecationWarning)
+       import nntplib
```

Comment:
Will this be global, or just for this particular file? If it is global, we should "tag" `testsetup`, `testcleanup`, and all `doctests` with a named group.

Reply:
As I understand things it should be local to this file, but I can test that.

Reply:
Turned out to be effectively global; fixed (I think :)).

Reply:
Ah, good to know! Also: nice hack :)